src/webob/acceptparse.py

```diff
@@ -686,10 +686,16 @@ def basic_filtering(self, language_tags):
         suggested as a matching scheme for the ``Accept-Language`` header in
         :rfc:`RFC 7231, section 5.3.5 <7231#section-5.3.5>`, and defined in
         :rfc:`RFC 4647, section 3.3.1 <4647#section-3.3.1>`. It filters the
-        tags in the `language_tags` argument, and returns a list of the tags
-        that match the language ranges in the header according to the Basic
-        Filtering matching scheme, in descending order of preference, together
-        with the qvalue of the range each tag matched.
+        tags in the `language_tags` argument and returns the ones that match
```

Comment:
Single backticks alone render in HTML markup as `<cite>`:

```html
<cite>language_tags</cite>
```

As an aside, I wish Sphinx had used `<code>` or `<pre>` instead, because `<cite>` is semantically incorrect. `¯\_(ツ)_/¯`

By default most web browsers use italics for styling `<cite>`. We could style the `<cite>` tags however we want, say as preformatted or monospace code, but italics are fine for now.

The page you linked uses a `:role:` preceding the single backticked item. You could try the `:any:` role preceding `language_tags` and see if magic happens or if the HTML markup changes to something not italicized. Else there might be a specific existing role under the [Python domain in Sphinx](http://www.sphinx-doc.org/en/stable/domains.html#the-python-domain). Finally we could define a custom role for parameters, if none of the existing ones are satisfactory, but I think that would be too much effort for very little or no benefit.